Makefile

```diff
@@ -0,0 +1,64 @@
+.PHONY: default
+default: help
+
+.PHONY: help
+help:
+	@echo "make help              Show this help message"
+	@echo "make dev               Run the pattern library application (TODO)"
+	@echo "make build             Build the package"
+	@echo "make lint              Run the code linter(s) and print any warnings"
+	@echo "make checkformatting   Check code formatting"
+	@echo "make format            Automatically format code"
+	@echo "make test              Run the unit tests once"
+	@echo "make sure              Make sure that the formatter, linter, tests, etc all pass"
+	@echo "make clean             Delete development artefacts (cached files, "
+	@echo "                       dependencies, etc)"
```

Comment:
Almost might be worth added a `make push` to map to the `yarn push` which will update the lib/ folder in linked references such as the client.

Reply:
Let's perhaps discuss this at the next frontend sync or other group chat? I can see arguments both ways.